site/src/components/sections/SectionInfo.vue

```diff
@@ -57,6 +62,10 @@ export default class SectionInfo extends Vue {
   get modalTitle(): string {
     return `Section Info: ${this.section.sec} - ${this.section.title}`;
   }
+
+  get course(): string {
```

Comment:
Could you rename this function to `courseCode` to be a bit more descriptive?

Reply:
Fixed